6.4.remember_remember.py

```diff
@@ -0,0 +1,26 @@
+from PIL import Image
+
+
+def decoding_an_image(path):
+    # Open the image
+    try:
+        img = Image.open(path)
+    except:
+        print("Wrong path!")
+        return
+    width = img.width
+    height = img.height
+
+    my_string = ''
+    # Goes over all pixels
+    for i in range(0, width):
+        for j in range(0, height):
+            # Check if pixel is black
+            if img.getpixel((i, j)) == 1:
+                # Convert the line number of black pixel to character by ASCII value
+                my_string += chr(j)
```

Comment:
1. Try to avoid i and j as well if there is a better name available.

2. Replace these two for loops with a list comprehention with two iterators - 
https://stackoverflow.com/questions/18551458/how-to-frame-two-for-loops-in-list-comprehension-python